tests/data_context/test_configuration_storage_pre_v013.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import os
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+
+def read_config_file_from_disk(config_filepath):
+    with open(config_filepath) as infile:
+        config_file = infile.read()
+        return config_file
+
+
+def test_preserve_comments_in_yml_after_adding_datasource(
+    data_context_parameterized_expectation_suite_no_checkpoint_store,
+):
+    ### THIS TEST FAILING IS A KNOWN ISSUE.
+    ### A TICKET IS OPEN
+
+    pytest.skip("KNOWN ISSUE")
+    #####
+    #
+    # KNOWN ISSUE: THIS DOES NOT FULLY PRESERVE WHITESPACE
+    # PROGRAMMATIC ADDITION MAY NOT BE PRESERVED IN PY2 AS WELL
+    # HOWEVER, GIVEN SHORT TIME TO EOL OF PY2, WE ARE WILLING TO ACCEPT THAT
+    #
+    #####
```

Comment:
I imagine this is a copy of another test, but since py2 is long deprecated could we remove this cruft?